website/blog/2022-06/00/Gardener-Community-Meeting-July.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Community Call - Gardener Extension Development
+linkTitle: Community Call - Gardener Extension Development
+newsSubtitle: June 17, 2022
+publishdate: 2022-06-17
+authors:
```

Comment:
Can we somehow include the speakers from the community call in the blog?
Either as authors or as part of the content (e.g. in the beginning)